### DISCUSSION TITLE
[Keyball61]左２キー及び右２キーの反応がありません。ご教授ください。

### DISCUSSION THREAD

**Discussion by cutiecyber** [Yowkees/keyball#483] [Q&A]
お世話になります。添付ファイルのとおり、左右２キーずつ反応のないキーがございます。キーソケット及びダイオードのはんだ付けや、付け替え等を行いたいと思っておりますが、対応するダイオードの位置が分からないため、ご教授願います。
もし、その前に取り組めることがございましたら、合わせてご指導願います。

![keyball61remap](https://github.com/Yowkees/keyball/assets/160136414/6e566182-ff05-4542-88cd-dc0052533a73)
![keyball61](https://github.com/Yowkees/keyball/assets/160136414/a810d0f0-d507-4d9b-bf5c-ea131fb17475)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

反応ないキーにつきまして、添付しました写真の色どうしのキーソケットとダイオードが対応しているものどうしになります。
キーソケットのはんだ付けが原因の場合もありますので、キーソケットのはんだ付けのやり直しも併せてご確認をよろしくお願いいたします。
![305083185-a810d0f0-d507-4d9b-bf5c-ea131fb17475](https://github.com/Yowkees/keyball/assets/143698349/ad73b176-4fad-49d4-a15f-1897d9cfb813)

**Reply by cutiecyber** [reply]
ご丁寧にご教示いただきありがとうございます。
早速取り組もうと、キースイッチを外したところ、なんと４つともスイッチの金具が曲がっており単純に刺さっておりませんでした。
伸ばして差し込んだところ、あっさりとすべてのキーが反応いたしました。
この度は、ご面倒をおかけして申し訳ありませんでした。
少しずつ、キーマップをいじり、使っていきたいと思います。

**Reply by Ryo624** [reply]
キースイッチの足折れでしたか！問題が解決して良かったです。

ぜひ末永くKeyball61をご愛用いただければ幸いです。
